frontend/src/app/modules/group/group-chat/group-chat.component.ts

```diff
@@ -73,6 +73,9 @@ export class GroupChatComponent extends BaseComponent implements OnInit, AfterVi
   }
 
   sendMessage(){
+    if (this.textOfMessage === ''){
```

Comment:
Could you disable the button instead of this?

Reply:
it's disabled, but there is actually an opportunity to send messages by pressing "enter"

Reply:
so this case is only for this option

Reply:
OK